spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -367,7 +364,7 @@ def expect_logger_msg(level, msg)
                    'mhvCorrelationId' => loa3_user.mhv_correlation_id,
                    'signIn' => { 'serviceName' => 'idme' },
                    'credential_used' => 'id_me',
-                   'expirationTime' => expire_at.iso8601(0))
+                   'expirationTime' => 30.minutes.from_now.iso8601(0))
```

Comment:
These *could* be wrapped in a `Timecop.freeze` block to get back to being _correct_ about this. The `iso8601(0)` function basically strips off seconds though, so unless it takes ~1 minute for this single spec to run, or we hit a ~1s run from :59 to :00 seconds (next minute) we'd be fine.

If we do need to wrap it, I would recommend/plan to use `travel_to` Rails helper instead

Reply:
I'd prefer to see it wrapped with `freeze_time { }` (`rails-5.2` alias of `travel_to Time.now`). It would probably be fine either way but might cause a headache to track down in the future. 

That said I think we should open a track of work to remove the basically deprecated `Timecop` dependency altogether. So I did that [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/2007)

Reply:
Yeah -- i did a quick pass with `freeze_time` and started down the "add it to the Rspec.config or just include here or what..." 

I would be HAPPY to help refactor this.